docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Device_operations/Synchronize_devices_with_Azure_IoT_Hub.md

```diff
@@ -37,7 +37,7 @@ In the import process, you will need a list of devices that you want to import a
         ![Importing devices](images/import_view.png "Importing devices")
         - If you already have a CSV file with devices for import, click **Browse** and select the file.
         - If you don't have a CSV with devices for import yet, click the **device template** link to download an empty template that you can fill in with your data:
-            - **DeviceId**	- provide your device ID. For LwM2M-enabled devices, device ID usually equals the endpoint name.
+            - **EndpointName** - provide EndpointName of your device.
```

Comment:
`provide endpoint name of your device` looks better

Reply:
done